rpy2/rlike/container.py

```diff
@@ -143,6 +143,7 @@ class TaggedList(list):
     """
 
     __tags: List[Optional[str]]
+    __tags = list()
```

Comment:
This creates a class-level list. The problem with this is that for some reason an instance of the class does not initialize its own instance-specific list it will be shared by all instances.